### ISSUE TITLE
SDP also contains session identifier

### ISSUE THREAD

**Issue by Worldwidebrine** [mdn/content#36500]
### MDN URL

https://developer.mozilla.org/en-US/docs/Glossary/SDP

### What information was incorrect, unhelpful, or incomplete?

> SDP (Session Description [Protocol](https://developer.mozilla.org/en-US/docs/Glossary/Protocol)) is the standard describing a [peer-to-peer](https://developer.mozilla.org/en-US/docs/Glossary/P2P) connection. SDP contains the [codec](https://developer.mozilla.org/en-US/docs/Glossary/Codec), source address, and timing information of audio and video.

### What did you expect to see?

Unique id is important, I guess?

I was trying to figure out if cache for signaling is possible, and I found information unique or always changes so far is just `sess-id`. Is this correct?

### Do you have any supporting links, references, or citations?

`<sess-id> is a numeric string such that the tuple of <username>, <sess-id>, <nettype>, <addrtype>, and <unicast-address> forms a globally unique identifier for the session. The method of <sess-id> allocation is up to the creating tool, but a timestamp, in seconds since January 1, 1900 UTC, is recommended to ensure uniqueness.`
https://www.rfc-editor.org/rfc/rfc8866.html

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/glossary/sdp`
* MDN URL: https://developer.mozilla.org/en-US/docs/Glossary/SDP
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/glossary/sdp/index.md
* Last commit: https://github.com/mdn/content/commit/ada5fa5ef15eadd44b549ecf906423b4a2092f34
* Document last modified: 2023-06-08T13:19:16.000Z

</details>

**Comment by wbamberg** [comment]
I don't think this page needs to list session ID. It's a glossary entry, not documentation for the protocol. I don't think it's realistic for it to include all the important parts of the format.

As a glossary page, this page needs to do three things:

- say what the item is at a high level
- say how it's related to/used in the web platform
- link to more details

That said, this isn't a very successful page on those terms:
- it doesn't give a great explanation of what it is (e.g. "describing a [peer-to-peer](https://developer.mozilla.org/en-US/docs/Glossary/P2P) connection" when the main point seems to be that it is about _media_)
- although it mentions WebRTC it doesn't say where in WebRTC is it used, and WebRTC is huge
- it doesn't link to https://datatracker.ietf.org/doc/html/rfc8866

**Comment by Josh-Cena** [comment]
@wbamberg Would you consider https://github.com/mdn/content/pull/44621 as sufficient for fixing this issue? It does fix your comment, but I still don't think there's any place where we actually document SDP (provided it is worth documenting and useful for readers).

**Comment by wbamberg** [comment]
It's an improvement, and yes we don't document SDP, we even say we don't (https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Protocols#sdp).

I think one thing that would still help would be to say where in WebRTC we use SDP. WebRTC is massive so just waving at it as a whole is not all that helpful.

I'm no expert on RTC but it looks like SDP is used in the [RTCSessionDescription](https://developer.mozilla.org/en-US/docs/Web/API/RTCSessionDescription), which gets set by [setRemoteDescription](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/setRemoteDescription) and [setLocalDescription](https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection/setLocalDescription) as part of [session negotiation](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Perfect_negotiation). So I think contextual links (i.e. not just bare links in "See also", but links accompanied by descriptions of what they are linking to and why) into those parts of the docs would help.

i.e. something like:

> SDP is used in WebRTC as part of [session negotiation](), as a way to describe the local and remote connections and how they are configured. The two WebRTC peers exchange [RTCSessionDescription]() objects, each of which uses SDP to describe the connection configuration options supported by that peer.
